src/components/IOUConfirmationList.js

```diff
@@ -30,11 +30,6 @@ const propTypes = {
     // IOU amount
     iouAmount: PropTypes.string.isRequired,
 
-    // Selected currency from the user
-    // Remove eslint disable after currency symbol is available
-    // eslint-disable-next-line react/no-unused-prop-types
-    selectedCurrency: PropTypes.string.isRequired,
-
```

Comment:
Actually, instead of removing the currency prop, could you ensure that we are passing currency symbol. Notice that the amount is £22, but shows $22 next to the participant.

![Screenshot 2021-05-06 at 12 41 26](https://user-images.githubusercontent.com/10736861/117292912-a4d22780-ae68-11eb-900e-cb38fad82bcb.png)

Reply:
+1